tests/Handler/CommandNameExtractor/ClassNameExtractorTest.php

```diff
@@ -12,7 +12,7 @@ class ClassNameExtractorTest extends TestCase
      */
     private $extractor;
 
-    protected function setUp()
+    protected function setUp(): void
```

Comment:
Can't do this on PHP 7.0. I think we should just change the PHP version constraint to `^7.1.3 || ^8.0`.

Reply:
Right... Working CI miss there...